src/test/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/cost/WheelchairCostCalculatorTest.java

```diff
@@ -0,0 +1,53 @@
+package org.opentripplanner.routing.algorithm.raptoradapter.transit.cost;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.model.WheelChairBoarding;
+import org.opentripplanner.routing.api.request.WheelchairAccessibilityFeature;
+import org.opentripplanner.routing.api.request.WheelchairAccessibilityRequest;
+import org.opentripplanner.transit.raptor.api.transit.RaptorTransferConstraint;
+
+public class WheelchairCostCalculatorTest {
+
+  final int UNKNOWN_ACCESSIBILITY_COST = 500;
+  final int INACCESSIBLE_TRIP_COST = 10000;
+
+  private final WheelchairCostCalculator subject = new WheelchairCostCalculator(
+    new DummyCostCalculator(),
```

Comment:
Instead of having to maintain the DummyCostCalculator, you could create a DefaultCostCalculator and use the existing TestTripSchedule:
```
  private static final int BOARD_COST_SEC = 5;
  private static final int TRANSFER_COST_SEC = 2;
  private static final double WAIT_RELUCTANCE_FACTOR = 0.5;

  private final DefaultCostCalculator delegateCalculator = new DefaultCostCalculator(
    BOARD_COST_SEC,
    TRANSFER_COST_SEC,
    WAIT_RELUCTANCE_FACTOR,
    null,
    null
  );
  private final tripBuilder = TestTripSchedule.schedule("12:00 12:01");
  private final baseBoardCost = delegateCalculator.boardingCost(true, 0, 5, 100, schedule, RaptorTransferConstraint.REGULAR_TRANSFER);
```

then to make you test independent of the `DefaultCostCalculator` implementation:

```
@Test
public void boardTripWithUnknownAccessibility() {
  var schedule = tripBuilder.withWheelChairBoarding(WheelChairBoarding.NO_INFORMATION).build();

  int result = calcBoardCost(schedule);
  int expected = baseBoardCost + RaptorCostConverter.toRaptorCost(UNKNOWN_ACCESSIBILITY_COST);

  assertEquals(expected, result);
}
```

Reply:
I implemented this according to your suggestion.